src/Messages.ts

```diff
@@ -306,9 +306,31 @@ export default class Messages {
     return `rgba(${r},${g},${b},${a})`
   }
 
+  private addDefaults(vars: MessageVariables): MessageVariables {
+    const kinds = this.getMessages().actionDefinitions || {}
+    const defaults = kinds[vars.__name__]
+
+    if (!defaults) {
+      return vars
+    }
+
+    function useDefaults(obj: MessageVariables, defaultValues: MessageVariables): MessageVariables {
+      for (const key in defaultValues) {
```

Comment:
nit: Consider changing to:
```suggestion
      for (const key of Object.keys(defaultValues)) {
```